core/storage/base_model/gae_models.py

```diff
@@ -62,13 +62,7 @@
     from mypy_imports import datastore_services
     from mypy_imports import transaction_services
 
-    BaseLogEntryAllowedCommitCmdTypes = Union[
-        Sequence[Mapping[str, change_domain.AcceptableChangeDictTypes]],
-        Mapping[str, change_domain.AcceptableChangeDictTypes],
-        None
-    ]
-
-    VersionedAllowedCommitCmdTypes = Sequence[
+    AllowedCommitCmdTypes = Sequence[
```

Comment:
Let's call this ListOfAllowedCommitCmds or similar? It's basically one type, not multiple types, and that type is a list of stuff.

Reply:
The name is changed to `AllowedCommitCmdsListType`, I'm emphasizing on `Type` word because it helps to separate type variables from normal variables. Also, I didn't use `ListOfAllowedCommitCmds` and `ListTypeOfAllowedCommitCmds` because it sounds like more as a sentence than a variable name. But if you have any concerns, I'm ok to change it to `ListOfAllowedCommitCmds`.